docs/source/conf.py

```diff
@@ -13,38 +13,88 @@
 
     sys.path.insert(0, os.path.abspath("../../src"))
 
-project = "SNAPRed"
-project_copyright = "2021, ORNL"
-author = "ORNL"
+import mock
```

Comment:
Change this from `import mock` to `import unittest.mock as mock`. Otherwise, this looks good!